### ISSUE TITLE
help

### ISSUE THREAD

**Issue by jovian-2wang** [BUPTLdy/MARTA-GAN#7]
Hello, I have adjusted tensflow and tenslayer many times in current  code, but it still can not success. I would like to ask you the correct version of tensflow and tenslayer.

**Comment by bianyuelin** [comment]
Hello, I am also troubled by the version problem. Have you solved your problem?

**Comment by JLC7** [comment]
Yeah. I am also having a lot of trouble. What versions do we use?

**Comment by bianyuelin** [comment]
tensorflow 1.4.0&nbsp; &nbsp;tensorlayer 1.4.1






------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"JLC"<notifications@github.com&gt;;
发送时间:&nbsp;2019年11月5日(星期二) 上午8:17
收件人:&nbsp;"BUPTLdy/MARTA-GAN"<MARTA-GAN@noreply.github.com&gt;;
抄送:&nbsp;"-273.15℃"<1779349527@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [BUPTLdy/MARTA-GAN] help (#7)




Yeah. I am also having a lot of trouble. What versions do we use?
